docs/developer-docs/multi-chain/bitcoin/using-btc/runes.mdx

```diff
@@ -0,0 +1,112 @@
+---
+keywords: [advanced, bitcoin, tutorial, btc, runes, schnorr, taproot]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Runes
+
+<MarkdownChipRow labels={["Advanced", "Bitcoin", "Tutorial"]} />
+
+## Overview
+
+Bitcoin Runes are a type of fungible asset deployed on the Bitcoin network. Runes are not reliant on the [Ordinals protocol](ordinals.mdx) like other Bitcoin asset standards such as BRC-20 and SRC-20. Runes are designed to be an efficient and simple asset that utilizes Bitcoin's UTXO model and the `OP_RETURN` opcode.
```

Comment:
It might be helpful to add that although Runes are not built on top of Ordinal theory. The protocol was created by the same inventor and is implemented in the same open source project ord.

Reply:
Added additional context